cmd/operator/main.go

```diff
@@ -111,7 +111,7 @@ func main() {
 
 func runCLI(logger *logrus.Logger, cfg *config.Config) {
 	for {
-		services, err := getTargetedServices(context.Background(), cfg.Targets)
```

Comment:
dedupe `context.Background()`s while at it.

ideally there should be 1 `context.Background()` per program